src/components/CommentComposer/CommentComposerProgress.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react'
+import { css } from 'glamor'
+
+const styles = {
+  circle: css({
+    transition: 'stroke-dashoffset 0.35s',
+    transform: 'rotate(-90deg)',
+    transformOrigin: '50% 50%'
+  })
+}
+
+
+class CommentComposerProgress extends Component {
+  constructor(props) {
+    super(props)
+    
+    const { radius, strokeWidth } = this.props
+    
+    this.normalizedRadius = radius - strokeWidth * 2
```

Comment:
should be `radius - strokeWidth / 2`. SVG strokes are 50% outside, 50% inside (may be configurable in the future). Currently this causes miss-alignment of the circle with the submit comment button below. Reduce the radius from 12 to 9 after changing this.